src/sentry/tasks/sentry_apps.py

```diff
@@ -274,29 +272,27 @@ def send_webhooks(installation, event, **kwargs):
 
         request_data = AppPlatformEvent(**kwargs)
 
-        buffer = SentryAppWebhookRequestsBuffer(installation.sentry_app)
-
-        try:
-            resp = safe_urlopen(
-                url=servicehook.sentry_app.webhook_url,
-                data=request_data.body,
-                headers=request_data.headers,
-                timeout=5,
-            )
-        except RequestException:
-            # Response code of 0 represents timeout
-            buffer.add_request(
-                response_code=0,
-                org_id=installation.organization_id,
-                event=event,
-                url=servicehook.sentry_app.webhook_url,
-            )
-            # Re-raise the exception because some of these tasks might retry on the exception
-            raise
-
-        buffer.add_request(
-            response_code=resp.status_code,
-            org_id=installation.organization_id,
-            event=event,
-            url=servicehook.sentry_app.webhook_url,
+        send_and_save_webhook_request(
+            servicehook.sentry_app.webhook_url, installation.sentry_app, request_data
         )
+
+
+def send_and_save_webhook_request(url, sentry_app, app_platform_event):
```

Comment:
@nolachen I'd like to be consistent about the order of arguments for some of these functions. For example, `url` before `sentry_app` here but in `send_and_save_request` it's after.

Reply:
@scefali Yeah I agree. Also if you know a better place (maybe like a utils file?) where this function could belong that might make more sense too

Reply:
@nolachen I think the current location is OK.